### PR TITLE
New package: bsnes-111

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

**Comment by obunden** [comment]
byuu released a fix for this release (v111.1) just now. Will you keep working at it? I was about to give making a new package for bsnes when I stumbled upon yours :)

**Comment by asiekierka** [comment]
Yes, but if you actually use bsnes regularly it might be wiser for you to take up the maintaining handle, if only because I might be slower to update.

**Comment by obunden** [comment]
I don't have much experience with this kind of thing so I don't know if I'm the right guy to be honest, not at the moment anyways. I use retroarch with snes9x/bsnes/higan cores in that order. I just thought that it would be nice to have have the standalone version for users that can't use higan because of slow computers. Tell me if you don't have time or if you don't want to work on it anymore and I will give it a try.

**Comment by asiekierka** [comment]
It appears byuu is still stabilizing version 111, so please wait with this for now.

File: srcpkgs/bsnes/template [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+make_install_args="prefix=${DESTDIR}/usr"
+hostmakedepends="unzip pkg-config"
+makedepends="gtk+-devel gtksourceview2-devel libgomp-devel
+ $(vopt_if sdl2 SDL2-devel)
+ $(vopt_if opengl MesaLib-devel) $(vopt_if xv libXv-devel)
+ $(vopt_if ao libao-devel) $(vopt_if alsa alsa-lib-devel)
+ $(vopt_if openal libopenal-devel) $(vopt_if pulseaudio pulseaudio-devel)"
+short_desc="SNES / SFC emulator"
+maintainer="Adrian Siekierka <kontakt@asie.pl>"
+license="GPL-3"
```
Comment (Johnnynator) [inline srcpkgs/bsnes/template]:
```suggestion
license="GPL-3.0-only"
```

**Comment by obunden** [comment]
Looks like v111.3 has been out for 5 days now so should be "stable" now?

**Pull request closed without merging**